### ISSUE TITLE
go 1.5.3 fails to build on Solaris

### ISSUE THREAD

**Issue by drook** [golang/go#14211]
go 1.5.3 fails building tests on Solaris.

Solaris:

````uname
# uname -a
SunOS hyperion 5.11 11.2 i86pc i386 i86pc
````

GCC:

````gcc
# gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/gcc/4.5/lib/gcc/i386-pc-solaris2.11/4.5.2/lto-wrapper
Target: i386-pc-solaris2.11
Configured with: /builds/hudson/workspace/nightly-update/build/i386/components/gcc45/gcc-4.5.2/configure CC=/ws/on11update-tools/SUNWspro/sunstudio12.1/bin/cc CXX=/ws/on11update-tools/SUNWspro/sunstudio12.1/bin/CC --prefix=/usr/gcc/4.5 --mandir=/usr/gcc/4.5/share/man --bindir=/usr/gcc/4.5/bin --libdir=/usr/gcc/4.5/lib --sbindir=/usr/gcc/4.5/sbin --infodir=/usr/gcc/4.5/share/info --libexecdir=/usr/gcc/4.5/lib --enable-languages=c,c++,fortran,objc --enable-shared --with-gmp-include=/usr/include/gmp --with-mpfr-include=/usr/include/mpfr --without-gnu-ld --with-ld=/usr/bin/ld --with-gnu-as --with-as=/usr/gnu/bin/as CFLAGS='-g -O2 '
Thread model: posix
gcc version 4.5.2 (GCC)
````

Environment:

````ENV
CFLAGS="-g -m64"
LDFLAGS="-g -m64"
CXXFLAGS="-g -m64"
````

Boostrap:

````bootstrap
GOOS=solaris
GOARCH=amd64
````

bootstrapped with go 1.4.3

Build log:

````building
[root@hyperion src]# ./all.bash
##### Building Go bootstrap tool.
cmd/dist

##### Building Go toolchain using /usr/local/go.
bootstrap/internal/obj
bootstrap/asm/internal/flags
bootstrap/compile/internal/big
bootstrap/internal/gcprog
bootstrap/internal/obj/arm
bootstrap/internal/obj/arm64
bootstrap/internal/obj/ppc64
bootstrap/internal/obj/x86
bootstrap/asm/internal/lex
bootstrap/link/internal/ld
bootstrap/compile/internal/gc
bootstrap/asm/internal/arch
bootstrap/asm/internal/asm
bootstrap/asm
bootstrap/link/internal/amd64
bootstrap/link/internal/arm
bootstrap/link/internal/arm64
bootstrap/link/internal/ppc64
bootstrap/link/internal/x86
bootstrap/link
bootstrap/compile/internal/amd64
bootstrap/compile/internal/arm
bootstrap/compile/internal/arm64
bootstrap/compile/internal/ppc64
bootstrap/compile/internal/x86
bootstrap/compile

##### Building go_bootstrap for host, solaris/amd64.
runtime
errors
sync/atomic
sync
internal/singleflight
io
unicode
unicode/utf8
unicode/utf16
bytes
math
strings
strconv
bufio
sort
container/heap
encoding/base64
syscall
internal/syscall/windows/registry
time
internal/syscall/windows
os
reflect
fmt
encoding
encoding/binary
encoding/json
flag
path/filepath
path
io/ioutil
log
regexp/syntax
regexp
go/token
go/scanner
go/ast
go/parser
os/exec
os/signal
net/url
text/template/parse
text/template
go/doc
go/build
hash
crypto
crypto/sha1
debug/dwarf
debug/elf
debug/macho
cmd/go

##### Building packages and commands for solaris/amd64.
runtime
errors
sync/atomic
unicode
unicode/utf8
math
container/list
container/ring
sort
crypto/subtle
encoding
unicode/utf16
image/color
runtime/race
cmd/vet/whitelist
sync
image/color/palette
container/heap
io
syscall
internal/singleflight
hash
crypto/cipher
runtime/trace
hash/crc32
hash/adler32
crypto/hmac
hash/crc64
hash/fnv
bytes
strings
time
bufio
text/tabwriter
path
html
strconv
math/rand
math/cmplx
compress/bzip2
os
reflect
regexp/syntax
crypto
crypto/aes
crypto/rc4
encoding/base64
encoding/base32
image
encoding/ascii85
crypto/sha512
crypto/md5
crypto/sha1
crypto/sha256
encoding/pem
image/internal/imageutil
path/filepath
os/signal
image/draw
image/jpeg
regexp
io/ioutil
os/exec
encoding/binary
fmt
cmd/pprof/internal/svg
crypto/des
index/suffixarray
flag
log
cmd/internal/rsc.io/arm/armasm
cmd/internal/rsc.io/x86/x86asm
debug/dwarf
debug/gosym
debug/plan9obj
archive/tar
compress/flate
compress/lzw
math/big
encoding/hex
go/token
database/sql/driver
encoding/csv
encoding/gob
encoding/json
encoding/xml
mime
mime/quotedprintable
go/scanner
net/http/internal
database/sql
cmd/internal/obj
net/url
text/template/parse
text/scanner
go/ast
image/gif
runtime/pprof
internal/trace
net/internal/socktest
archive/zip
compress/gzip
compress/zlib
testing
runtime/debug
debug/elf
debug/macho
debug/pe
image/png
testing/iotest
testing/quick
cmd/asm/internal/flags
text/template
cmd/compile/internal/big
cmd/internal/gcprog
cmd/dist
cmd/pack
crypto/dsa
crypto/elliptic
cmd/internal/goobj
encoding/asn1
crypto/rand
go/constant
cmd/internal/obj/arm
internal/testenv
cmd/internal/obj/arm64
cmd/internal/obj/ppc64
go/parser
go/printer
cmd/internal/obj/x86
cmd/asm/internal/lex
crypto/rsa
cmd/pprof/internal/profile
cmd/internal/objfile
cmd/link/internal/ld
crypto/ecdsa
crypto/x509/pkix
cmd/pprof/internal/tempfile
go/doc
html/template
go/types
cmd/addr2line
cmd/nm
cmd/objdump
cmd/compile/internal/gc
cmd/cgo
internal/format
go/format
cmd/gofmt
cmd/fix
cmd/yacc
cmd/pprof/internal/plugin
go/build
cmd/pprof/internal/symbolizer
cmd/asm/internal/arch
cmd/pprof/internal/symbolz
cmd/pprof/internal/report
cmd/asm/internal/asm
cmd/doc
cmd/cover
cmd/asm
cmd/pprof/internal/commands
cmd/pprof/internal/driver
go/internal/gccgoimporter
cmd/api
go/internal/gcimporter
cmd/link/internal/arm64
cmd/link/internal/amd64
cmd/link/internal/ppc64
cmd/link/internal/arm
cmd/link/internal/x86
go/importer
cmd/link
cmd/vet
runtime/cgo
net
os/user
cmd/compile/internal/amd64
cmd/compile/internal/arm
cmd/compile/internal/arm64
cmd/compile/internal/ppc64
cmd/compile/internal/x86
cmd/compile
crypto/x509
net/textproto
log/syslog
mime/multipart
net/mail
crypto/tls
net/http
net/smtp
net/http/pprof
net/http/cgi
net/http/httputil
net/http/httptest
net/rpc
expvar
net/http/cookiejar
cmd/go
cmd/pprof/internal/fetch
cmd/trace
cmd/pprof
net/http/fcgi
net/rpc/jsonrpc


##### Testing packages.
ok      archive/tar     0.056s
ok      archive/zip     1.144s
ok      bufio   0.157s
ok      bytes   0.298s
ok      compress/bzip2  0.189s
ok      compress/flate  1.370s
ok      compress/gzip   0.075s
ok      compress/lzw    0.156s
ok      compress/zlib   2.473s
ok      container/heap  0.025s
ok      container/list  0.017s
ok      container/ring  0.033s
ok      crypto/aes      0.053s
ok      crypto/cipher   0.047s
ok      crypto/des      0.036s
ok      crypto/dsa      0.031s
ok      crypto/ecdsa    0.103s
ok      crypto/elliptic 0.082s
ok      crypto/hmac     0.030s
ok      crypto/md5      0.025s
ok      crypto/rand     0.048s
ok      crypto/rc4      0.218s
ok      crypto/rsa      0.234s
ok      crypto/sha1     0.024s
ok      crypto/sha256   0.023s
ok      crypto/sha512   0.026s
ok      crypto/subtle   0.025s
ok      crypto/tls      1.807s
ok      crypto/x509     2.787s
ok      database/sql    0.097s
ok      database/sql/driver     0.059s
ok      debug/dwarf     0.033s
ok      debug/elf       0.068s
ok      debug/gosym     0.993s
ok      debug/macho     0.032s
ok      debug/pe        0.069s
ok      debug/plan9obj  0.025s
ok      encoding/ascii85        0.023s
ok      encoding/asn1   0.039s
ok      encoding/base32 0.024s
ok      encoding/base64 0.021s
ok      encoding/binary 0.030s
ok      encoding/csv    0.019s
ok      encoding/gob    0.110s
ok      encoding/hex    0.030s
ok      encoding/json   0.407s
ok      encoding/pem    0.104s
ok      encoding/xml    0.328s
ok      errors  0.019s
ok      expvar  0.155s
ok      flag    0.024s
ok      fmt     0.322s
ok      go/ast  0.034s
ok      go/build        1.170s
ok      go/constant     0.020s
ok      go/doc  0.255s
ok      go/format       0.045s
ok      go/internal/gccgoimporter       0.136s
ok      go/internal/gcimporter  1.195s
ok      go/parser       0.210s
ok      go/printer      0.584s
ok      go/scanner      0.022s
ok      go/token        0.084s
ok      go/types        3.127s
ok      hash/adler32    0.098s
ok      hash/crc32      0.018s
ok      hash/crc64      0.024s
ok      hash/fnv        0.018s
ok      html    0.044s
ok      html/template   0.106s
ok      image   0.316s
ok      image/color     0.183s
ok      image/draw      0.202s
ok      image/gif       0.303s
ok      image/jpeg      0.453s
ok      image/png       0.134s
ok      index/suffixarray       0.058s
ok      internal/singleflight   0.049s
ok      internal/trace  0.024s
ok      io      0.046s
ok      io/ioutil       0.037s
ok      log     0.025s
ok      log/syslog      2.041s
ok      math    0.020s
ok      math/big        1.346s
ok      math/cmplx      0.038s
ok      math/rand       0.601s
ok      mime    0.029s
ok      mime/multipart  0.321s
ok      mime/quotedprintable    0.399s
ok      net     3.267s
ok      net/http        8.468s
ok      net/http/cgi    1.854s
ok      net/http/cookiejar      0.028s
ok      net/http/fcgi   0.026s
ok      net/http/httptest       0.037s
ok      net/http/httputil       0.042s
ok      net/http/internal       0.038s
ok      net/internal/socktest   0.269s
ok      net/mail        0.052s
ok      net/rpc 0.068s
ok      net/rpc/jsonrpc 0.095s
ok      net/smtp        0.076s
ok      net/textproto   0.025s
ok      net/url 0.030s
ok      os      0.887s
ok      os/exec 1.672s
ok      os/signal       3.452s
ok      os/user 0.129s
ok      path    0.020s
ok      path/filepath   0.185s
ok      reflect 0.251s
ok      regexp  0.493s
ok      regexp/syntax   1.021s
--- FAIL: TestCgoCrashHandler (4.99s)
        crash_test.go:85: building source: exit status 2
                # _/tmp/go-build137264614
                /home/emz/src/go-go1.5.3/pkg/tool/solaris_amd64/link: running gcc failed: exit status 1
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-154141133/000000.o: section: [10].rela.debug_info: offset: 0x15: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-154141133/000000.o: section: [10].rela.debug_info: offset: 0x1d: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-154141133/000000.o: section: [10].rela.debug_info: offset: 0xa1: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-154141133/000000.o: section: [10].rela.debug_info: offset: 0xa9: relocation requires reference symbol
                ld: fatal: symbol referencing errors
                collect2: ld returned 1 exit status

--- FAIL: TestCgoSignalDeadlock (2.18s)
        crash_test.go:85: building source: exit status 2
                # _/tmp/go-build228811789
                /home/emz/src/go-go1.5.3/pkg/tool/solaris_amd64/link: running gcc failed: exit status 1
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-724713460/000000.o: section: [10].rela.debug_info: offset: 0x15: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-724713460/000000.o: section: [10].rela.debug_info: offset: 0x1d: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-724713460/000000.o: section: [10].rela.debug_info: offset: 0xa1: relocation requires reference symbol
                ld: fatal: relocation error: R_AMD64_64: file /tmp/go-link-724713460/000000.o: section: [10].rela.debug_info: offset: 0xa9: relocation requires reference symbol
                ld: fatal: symbol referencing errors
                collect2: ld returned 1 exit status

FAIL
FAIL    runtime 38.544s
ok      runtime/debug   0.042s
ok      runtime/pprof   8.075s
--- FAIL: TestTraceSymbolize (0.11s)
        trace_stack_test.go:273: did not match event GoSysCall at syscall.read:0
        trace_stack_test.go:274: seen the following events of this type:
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:42
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).listenStream:156
        trace_stack_test.go:280:   net.socket:76
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:76
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:76
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.ListenTCP:329
        trace_stack_test.go:280:   net.Listen:395
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:81
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:92
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:92
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).accept:400
        trace_stack_test.go:280:   net.(*TCPListener).AcceptTCP:254
        trace_stack_test.go:280:   net.(*TCPListener).Accept:264
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize.func10:86
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   os.(*File).Read:95
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize.func11:101
        trace_stack_test.go:282: ---
        trace_stack_test.go:282: ---
        trace_stack_test.go:282: ---
        trace_stack_test.go:282: ---
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:38
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:42
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).dial:137
        trace_stack_test.go:280:   net.socket:89
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).dial:137
        trace_stack_test.go:280:   net.socket:89
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.socket:89
        trace_stack_test.go:280:   net.internetSocket:160
        trace_stack_test.go:280:   net.dialTCP:171
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.newTCPConn:51
        trace_stack_test.go:280:   net.dialTCP:207
        trace_stack_test.go:280:   net.dialSingle:364
        trace_stack_test.go:280:   net.dialSerial.func1:336
        trace_stack_test.go:280:   net.dial:40
        trace_stack_test.go:280:   net.dialSerial:338
        trace_stack_test.go:280:   net.(*Dialer).Dial:232
        trace_stack_test.go:280:   net.Dial:186
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:118
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).Close:199
        trace_stack_test.go:280:   net.(*conn).Close:196
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:122
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   os.(*File).Write:139
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize:124
        trace_stack_test.go:280:   testing.tRunner:456
        trace_stack_test.go:282: ---
        trace_stack_test.go:280:   net.(*netFD).accept:400
        trace_stack_test.go:280:   net.(*TCPListener).AcceptTCP:254
        trace_stack_test.go:280:   net.(*TCPListener).Accept:264
        trace_stack_test.go:280:   runtime/trace_test.TestTraceSymbolize.func10:86
        trace_stack_test.go:282: ---
FAIL
FAIL    runtime/trace   7.526s
ok      sort    0.189s
ok      strconv 1.283s
ok      strings 0.211s
ok      sync    0.207s
ok      sync/atomic     4.466s
ok      syscall 0.177s
ok      testing 1.219s
ok      testing/quick   0.146s
ok      text/scanner    0.025s
ok      text/tabwriter  0.046s
ok      text/template   0.180s
ok      text/template/parse     0.051s
ok      time    3.310s
ok      unicode 0.032s
ok      unicode/utf16   0.031s
ok      unicode/utf8    0.030s
ok      cmd/addr2line   3.608s
ok      cmd/api 0.035s
ok      cmd/asm/internal/asm    0.075s
ok      cmd/asm/internal/lex    0.025s
ok      cmd/compile/internal/big        2.836s
ok      cmd/cover       4.808s
ok      cmd/doc 0.063s
ok      cmd/fix 0.053s
--- FAIL: TestCgoHandlesWlORIGIN (0.50s)
        go_test.go:251: running testgo [build origin]
        go_test.go:270: standard error:
        go_test.go:271: # origin
                ld: fatal: option '-rpath$ORIGIN' is incompatible with building a relocatable object
                collect2: ld returned 1 exit status

        go_test.go:280: go [build origin] failed unexpectedly: exit status 2
FAIL
FAIL    cmd/go  47.011s
ok      cmd/gofmt       0.109s
ok      cmd/internal/goobj      0.022s
ok      cmd/internal/obj        0.019s
ok      cmd/internal/obj/x86    0.115s
ok      cmd/internal/rsc.io/arm/armasm  0.025s
ok      cmd/internal/rsc.io/x86/x86asm  0.335s
ok      cmd/nm  2.335s
ok      cmd/objdump     5.601s
ok      cmd/pack        4.709s
ok      cmd/pprof/internal/profile      0.020s
ok      cmd/vet 8.419s
2016/02/03 19:41:06 Failed: exit status 1
````

**Comment by bradfitz** [comment]
Oh, this is for Go 1.5.3. We are not touching the Go 1.5.x branch at this point. Please try Go 1.6rc1 or 1.6rc2 coming out later today.

**Comment by drook** [comment]
1.6rc1 builds and runs tests just fine.

**Comment by bradfitz** [comment]
Then we're good. Thanks.

**Comment by binarycrusader** [comment]
FYI:

The TestCgoCrashHandler error looks familiar, but I can't quite remember which bug fix resolved that.

The TestTraceSymbolize failure was due to bug #12056 and was fixed for Go 1.6.

The -rpath$ORIGIN failure was due to bug #12115 and was fixed for Go 1.6.
